string[6] joint_names
float32[6] p
float32[6] i
float32[6] d